.github/actions/deploy/deploy.py

```
@@ -0,0 +1,1183 @@
+        from tests.sh in the operator repository.
+        """
+        if not self.args.deploy_external_db:
+            return
+
+        print(
+            '🔐 Applying MariaDB and MinIO Secrets and ConfigMaps to external namespace...'
+        )
+
+        if not self.operator_repo_path:
+            raise ValueError(
+                'Operator repository not cloned for external pre-requisites')
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)